device/sycl/src/clusterization/measurement_creation.hpp

```diff
@@ -12,15 +12,12 @@
 #include "traccc/edm/cluster.hpp"
 #include "traccc/edm/measurement.hpp"
 
-// Vecmem include(s).
-#include <vecmem/memory/memory_resource.hpp>
-
 namespace traccc::sycl {
 
 /// Forward decleration of measurement creation kernel
 ///
 void measurement_creation(measurement_container_view measurements_view,
                           cluster_container_view clusters_view,
-                          const unsigned int& range, queue_wrapper queue);
+                          const std::size_t& n_clusters, queue_wrapper queue);
```

Comment:
Why do you need to pass this size as a separate variable? :thinking: Isn't this the same as `clusters_view.m_size`?

Reply:
Alright, I changed it to `clusters_view.headers.size()`